router/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM golang:1.10-alpine AS builder
-
+RUN apk add --no-cache g++
```

Comment:
I can't think why we would need this?

Reply:
It fails while building the docker image otherwise

Reply:
That was happening only on ARM64 though.